web/app/js/components/ResourceDetail.jsx

```diff
@@ -227,6 +230,7 @@ export class ResourceDetailBase extends React.Component {
           upstreamMetrics,
           downstreamMetrics,
           edges,
+          resourceDefinition, // eslint-disable-line react/no-unused-state
```

Comment:
to clarify, is this `eslint-disable` because we are fetching the resourceDefinition but not actually displaying it anywhere?

Reply:
Yes, I added it only to avoid linter error due to `resourceDefinition` is not being used anywhere.